lib/mix/lib/mix.ex

```diff
@@ -161,6 +161,30 @@ defmodule Mix do
   Note aliases do not show up on `mix help`.
   Aliases defined in the current project do not affect its dependencies and aliases defined in dependencies are not accessible from the current project.
 
+  Aliases can be used very powerfully to also run elixir scripts and bash commands, for example:
```

Comment:
Checking the documentation I think it should be "Elixir"

Reply:
[Here is an example in the documentation](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/float.ex#L21)